src/cswinrt/main.cpp

```diff
@@ -94,12 +94,16 @@ Where <spec> is one or more of:
 
         settings.verbose = args.exists("verbose");
         auto target = args.value("target");
-        if (!target.empty() && target != "netstandard2.0" && !starts_with(target, "net5.0") && !starts_with(target, "net6.0") && !starts_with(target, "net7.0") && !starts_with(target, "net8.0"))
+        if (!target.empty() && target != "netstandard2.0" && !starts_with(target, "net8.0"))
```

Comment:
I was debating whether we wanted to do this given for some reason we left net5 before which I don't recall why.  It was probably to stage dropping support.  But due to the other simplifications to it led to in code_writers, it is probably fine.